units/systemd-bsod.service.in

```diff
@@ -0,0 +1,21 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Displays emergency message full screen.
+Documentation=man:systemd-bsod(8)
+ConditionVirtualization=no
+AssertPathExists=/etc/initrd-release
```

Comment:
this one should be fine to also run in the main system actually, please drop this assert

Reply:
Alright sir.
Thank you so much for showing up today. I have smiles all over my face.